CONTRIBUTING.md

```diff
@@ -59,3 +59,23 @@ Pull requests are subject to:
 
 - approval by one or more [maintainers](https://github.com/orgs/opctl/teams/maintainers/members)
 - the [build](.opspec/build) op continuing to run with a successful outcome
+
+# Local development with native go tooling
+
+First, ensure dependencies are installed
+
+- [go 1.15](https://golang.org/doc/go1.15)
```

Comment:
is it reasonable to refer to the compile op for go version (or alternatively store go version somewhere and reference that in all the ops) to keep a single source of truth? Otherwise I think it might get hard/error prone to track and update all the places we're one off documenting versions of things

Reply:
I can refer to the version in the go mod file, as that's the main thing go uses - there's about 9 different ops that refer to the go version.

Reply:
I like that. In the ops we could parse that file and grab version but seems overkill.. something to consider anyways